services/galley/test/integration/API/Teams.hs

```diff
@@ -210,6 +214,34 @@ testListTeamMembersTruncated = do
       "member list does not indicate that there are more members"
       (listFromServer ^. teamMemberListHasMore)
 
+testListTeamMembersDefaultLimitByIds :: TestM ()
+testListTeamMembersDefaultLimitByIds = do
+  (owner, tid, [member1, member2]) <- Util.createBindingTeamWithNMembers 2
+  check owner tid [owner, member1, member2]
+  check owner tid [member1, member2]
+  check owner tid [member1]
+  check owner tid [] -- a bit silly, but hey.
```

Comment:
How should the endpoint behave when you also ask for a non-existing user ID. Or *only* for non-existing ones? Or if your `maxResults`  is above the truncation limit?
We could test for some of these cases, too. (imho, mostly the first one case would be useful, maybe also the second)

Reply:
good idea.  31627edc4f5ebf9eaafc33a2c98d558ce4f1f2a6

I had some truncation errors in there, too, but then I realized that's tested in the next case, so I removed them.